### ISSUE TITLE
Modals Accessibility Issue

### ISSUE THREAD

**Issue by octoper** [laravel/jetstream#279]
- Jetstream Version: 1.3.0
- Jetstream Stack: Inertia & Livewire
- Laravel Version: 8.6.0
- PHP Version: 7.4.3

### Description:
When you open a modal, an overlay added and the focus moves to the input as expected.
But the focus doesn't trapped inside the modal or closes the modal when the focus is over the last element in it when tabbing, it's just moves to the next element on the whole page but the modal is still open.
This is not accessible to a lot of users not even those with real accessibility problems.

### Steps To Reproduce:
Just open a modal and click `Tab` and after the last element the focus moves outside the modal without even closing it.

**Comment by driesvints** [comment]
Feel free to send in a pr. Thanks
